analyzer/src/main/kotlin/managers/Maven.kt

```diff
@@ -96,7 +96,7 @@ class Maven(
     fun enableSbtMode() = also { sbtMode = true }
 
     override fun beforeResolution(definitionFiles: List<File>) {
-        val projectBuilder = mvn.container.lookup(ProjectBuilder::class.java, "default")
+        val projectBuilder = mvn.containerLookup<ProjectBuilder>()
```

Comment:
Commit message: Please use [imperative mood](https://github.com/oss-review-toolkit/ort/blob/master/CONTRIBUTING.md#git-history), like

    Simplify the access to objects in the Maven Plexus container

(and also no dot at the end of the title).